src/Me/components/Select/Select.test.js

```diff
@@ -45,8 +45,8 @@ describe('Select component', () => {
       />
     );
 
-    expect(getByText('one')).toBeInTheDocument();
-    expect(getByText('two')).toBeInTheDocument();
-    expect(getByText('Reached max items')).toBeInTheDocument();
+    getByText('one');
```

Comment:
I like more the `expect / to be` shape :) I find it more clear to read.

Reply:
I usualy prefer less code but its a valid point so fair enough (and you are the boss).

Reply:
> I usualy prefer less code

Not always of course

> and you are the boss

Don't ever say this again 😱

Reply:
I promise not to boss!